reporter/src/funTest/assets/evaluated-model-test-expected-statistics.yml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Commit message nits:
s/For now containing only summary statistics./For now this contains only summary statistics./
s/accross/across/
s/and global and local/as well as global and local/

Reply:
Done.